spec/features/schedule_spec.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+def expect_scheduled(event, time)
```

Comment:
This sounds like a shared example to me...

https://rspec.info/features/3-12/rspec-core/example-groups/shared-examples/